agent/worker.go

```diff
@@ -344,6 +357,49 @@ func reconcileSecrets(ctx context.Context, w *worker, assignments []*api.Assignm
 	return nil
 }
 
+func reconcileConfigs(ctx context.Context, w *worker, assignments []*api.AssignmentChange, fullSnapshot bool) error {
```

Comment:
Since the code is exactly the same, it would be more generic (receiving an `interface{}`)